frontend/src/metabase/query_builder/components/DataSelector/DataSelector.jsx

```diff
@@ -1009,10 +1015,10 @@ export class UnconnectedDataSelector extends Component {
     }
     if (!selectedDataBucketId) {
       return [
-        "card",
         "dataset",
         "table",
         ...(this.props.canSelectMetric ? ["metric"] : []),
+        ...(this.props.canSelectSavedQuestion ? ["card"] : []),
```

Comment:
there are only 2 places `canSelectMetric` is used in this file, and these are 2 places that I added `canSelectSavedQuestion` to.